ci-operator/step-registry/hypershift/mce/power/create/ibmcloud-workers/infra/hypershift-mce-power-create-ibmcloud-workers-infra-commands.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+
+set -x
+
+# Agent hosted cluster configs
+HOSTED_CLUSTER_NAME="$(printf $PROW_JOB_ID|sha256sum|cut -c-20)"
+export HOSTED_CLUSTER_NAME
+export HOSTED_CONTROL_PLANE_NAMESPACE="${CLUSTERS_NAMESPACE}-${HOSTED_CLUSTER_NAME}"
+
+# Installing generic required tools
+echo "$(date) Installing required tools"
+mkdir /tmp/ibm_cloud_cli
+curl --output /tmp/IBM_CLOUD_CLI_amd64.tar.gz https://download.clis.cloud.ibm.com/ibm-cloud-cli/2.16.1/IBM_Cloud_CLI_2.16.1_amd64.tar.gz
+tar xvzf /tmp/IBM_CLOUD_CLI_amd64.tar.gz -C /tmp/ibm_cloud_cli
+export PATH=${PATH}:/tmp/ibm_cloud_cli/Bluemix_CLI/bin
+mkdir /tmp/bin
+curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /tmp/bin/jq && chmod +x /tmp/bin/jq
+export PATH=$PATH:/tmp/bin
+
+set_x86_configs() {
+  # VPC VSI(Virtual Server Instance) configs
+  VPC_VSI_NAME="x86-${HOSTED_CLUSTER_NAME}-worker"
+  PROFILE_NAME="bx2-2x8"
+  IMAGE_ID="r034-33250be2-61bb-4837-9cd5-6ef83c2ccb2c"
```

Comment:
What's that image id?

Reply:
It is the ID of the image present in IBM Cloud which will be used to create our VM.
Have added comments in the code regarding these specs.
Thanks!